apis/core/lib/domain/table/create.ts

```diff
@@ -26,13 +27,15 @@ interface CreateTableCommand {
   resource: GraphPointer
   store: ResourceStore
   dimensionMetadataQueries?: Pick<typeof DimensionMetadataQueries, 'getDimensionMetaDataCollection'>
+  tableQueries?: Pick<typeof TableQueries, 'getCubeTable'>
```

Comment:
As in other functions in `lib/domain/`, this SPARQL query is injected as an optional function call